src/odemis/driver/tmcm.py

```diff
@@ -418,6 +421,44 @@ def __init__(self, name, role, port, axes, ustepsize, address=None,
             self._updateTemperatureVA() # make sure the temperature is correct
             self._temp_timer.start()
 
+    def _update_ref(self):
+        """
+        It updates the global parameter AREF_USER_VAR with the axes that are referenced
+        """
+        # AREF_USER_VAR is a general purpose 32-bit variable in bank 2. It is used to store the axes that are already referenced.
+        # Up to 256 variables are available, only the first 56 can be stored permanently in EEPROM. #117 is randomly chosen between 56-255.
+        # In case the TMCM controller is turned off, the global parameter is automatically reset.
+        aref = 0
+        for n, i in self._name_to_axis.items():
+            if self.referenced.value[n]:
+                aref |= 1 << i
+        # special format: one bit per axis + inverted bit on the high 2 bytes to double check
+        aref |= ~aref << 16
+        self.SetGlobalParam(2, AREF_USER_VAR, aref)
+
+    def _is_already_ref(self, an):
```

Comment:
Please use `axis` as argument name, as in the other functions. In some other places we use `an`, but as "axis name". So it's a little confusing.